governance-server/src/logic.js

```diff
@@ -32,7 +32,7 @@ module.exports = class Logic {
             const delegationsScoreWeight = 0.5;
 
             const height = await this.aeternity.height();
-            const polls = (await this.aeternity.polls())
+            const polls = Object.entries(await this.aeternity.polls())
```

Comment:
are we not just returning an array from the polls function?

Reply:
good catch, leftover from initial refactor try